### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Daniil-Pankieiev**

### CODE REVIEW

**Review by SelFisher1488 — APPROVED** [review]
impressive job!

File: cinema/tests/test_movie_api.py [hunk 1]
```diff
@@ -157,3 +156,145 @@ def test_image_url_is_shown_on_movie_session_detail(self):
...
+        movie_with_genres = sample_movie()
+        movie_with_genres.genres.add(genre2, genre)
+        actor = sample_actor()
+        movie_with_actor = sample_movie()
+        movie_with_actor.actors.add(actor)
+        result = self.client.get(MOVIE_URL)
+        movies = Movie.objects.all()
+        serializer = MovieListSerializer(movies, many=True)
+        self.assertEqual(result.status_code, status.HTTP_200_OK)
+        self.assertEqual(result.data, serializer.data)
```
Comment (LiudmylaKulinchenko) [inline cinema/tests/test_movie_api.py:191]: add empty lines between genres and movies or try to place more logic in one line. It will make more readable

Comment (LiudmylaKulinchenko) [reply cinema/tests/test_movie_api.py:191]: fix in all tests

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

**Review by LiudmylaKulinchenko — APPROVED** [review]
Well done 🍬

**Pull request closed without merging**